service/host.go

```diff
@@ -92,6 +94,8 @@ func (uis *UIServer) hostsPage(w http.ResponseWriter, r *http.Request) {
 }
 
 func (uis *UIServer) modifyHost(w http.ResponseWriter, r *http.Request) {
+	env := evergreen.GetEnvironment()
+	queue := env.RemoteQueue()
```

Comment:
I would use uis.queue instead

Reply:
we should make sure that this is actually the remote queue, but yes.